controller/error_pages.rst

```diff
@@ -88,9 +94,6 @@ A typical project that returns HTML and JSON pages might look like this:
              ├─ error404.html.twig
              ├─ error403.html.twig
              ├─ error.html.twig      # All other HTML errors (including 500)
```

Comment:
Since the last option's where removed, this file tree is not correct anymore. Shouldn't this be changed to this:
```
├─ error404.html.twig
├─ error403.html.twig
└─ error.html.twig      # All other HTML errors (including 500)
```

Reply:
I didn't understand what you mean, would you please elaborate a little more?

Reply:
I meant that the last item in the file tree should be prefixed with these characters `└─` instead of these`├─`. Since this was the case when the `.json.twig` files where still there.

Reply:
Oh! correct.